sal_interfaces/SALSubsystems.xml

```diff
@@ -75,55 +75,55 @@
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Zaber</VendorContact>
   <Simulator>No</Simulator>
-  <Configuration>No</Configuration>
+  <Configuration>Not Configurable</Configuration>
 </SALSubsystem>
 
 <SALSubsystem>
   <Name>MTMount</Name>
   <Description>CSC for the Main Telescope Mount</Description>
-  <IndexEnumeration></IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Russell Owen</ActiveDevelopers>
   <Github>https://github.com/lsst-ts/ts_MTMount</Github>
-  <JenkinsTestResults>NA</JenkinsTestResults>
+  <JenkinsTestResults>Not Available</JenkinsTestResults>
   <RubinObsContact>Shawn Calahan</RubinObsContact>
   <CSCDocs>https://ts-mtmount.lsst.io</CSCDocs>
   <ProductOwner>Sandrine Thomas</ProductOwner>
-  <RelatedDocuments>LTS-150</RelatedDocuments>
+  <RelatedDocuments>LTS-103, LTS-150, LTS-159</RelatedDocuments>
   <SoftwareLanguage>Labview</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Tekniker</VendorContact>
   <Simulator>No</Simulator>
```

Comment:
This field should be updated.  Though I'm not sure to what.  Regardless, `No` is not a valid value for this field.  If only a shrug emoji was allowed...

Reply:
We are awaiting a response from another developer.  We'll handle this issue in a separate PR.